robocorp-code/vscode-client/src/extension.ts

```diff
@@ -324,6 +326,102 @@ async function cloudLogoutAndRefresh() {
     refreshCloudTreeView();
 }
 
+async function ensureConvertBundle() {
+    const url = "https://downloads.robocorp.com/converter/latest/bundle.js";
+    let location = getExtensionRelativeFile("../../vscode-client/out/converterBundle.js", false);
+    if (!(await fileExists(location))) {
+        await window.withProgress(
+            {
+                location: vscode.ProgressLocation.Notification,
+                title: "Downloading converter bundle",
+                cancellable: false,
+            },
+            async (
+                progress: vscode.Progress<{ message?: string; increment?: number }>,
+                token: vscode.CancellationToken
+            ): Promise<string | undefined> => await download(url, progress, token, location)
+        );
+    }
+    return location;
+}
+
+async function convertProject() {
+    const DEFAULT_ERROR_MSG = `
+            Could not convert project.
+            Please check the output logs for more details.
+            `;
+
+    const DEFAULT_ERROR_STATUS = "Error while converting project.";
+
+    const converterLocation = await ensureConvertBundle();
+    try {
+        const converterBundle = require(converterLocation);
+        window.setStatusBarMessage("Project conversion has started...");
+
+        // let the user decide what type of project will be converted
+        const vendorMap = {
+            "Blue Prism": "blueprism",
+            "UiPath": "uipath",
+            "Automation Anywhere 360": "a360",
+        };
+        const items = Object.keys(vendorMap);
+        let selectedItem = await window.showQuickPick(items, {
+            "placeHolder": "Please select vendor of current project",
+            "canPickMany": false,
+            "ignoreFocusOut": false,
+        });
+        window.setStatusBarMessage(`Selected project type: ${selectedItem}`);
```

Comment:
I think you shouldn't set `window.setStatusBarMessage` at all in this method (if the user just activated the action or pressed esc, etc, I don't think we should notify anything to him in the status bar).
-- the calls to information/error when done should already suffice.

You can use the `OUTPUT_CHANNEL` if you want to log something though (it will then be shown in `OUTPUT > Robocorp Code`), but I'm not sure we even need to log these (unless something went bad).